### DISCUSSION TITLE
Report Issue on PIM 11

### DISCUSSION THREAD

**Discussion by rushitkhunt1996** [pimcore/pimcore#15730] [General]
Hello,
We installed the report bundle on PIM 11. after successfully installed I created a report and clicked on the save button, now my admin panel is not working it's throwing an error.
![image](https://github.com/pimcore/pimcore/assets/60934941/de739d9a-9bc2-4ca7-8880-f36293021ad9)

![image](https://github.com/pimcore/pimcore/assets/60934941/26c09be6-0bd8-4d9f-8d7d-c73ce2de75cb)
If we remove this from test.yaml file which is automatically created from the report then the admin panel is working fine but the report is not there bcz we removed the configuration from yaml file.

can someone guide me i did something wrong or what?

**Comment by blankse** [comment]
@rushitkhunt1996 
See the update notes:
https://pimcore.com/docs/platform/Pimcore/Installation_and_Upgrade/Updating_Pimcore/V10_to_V11/#configuration-storage-settings

You have to move the settings to `pimcore_custom_reports`

**Reply by rushitkhunt1996** [reply]
yes, it's working. thanks @blankse
